src/js/points.js

```diff
@@ -31,8 +31,7 @@ var Points = function Points(settings) {
     canvas.className += ' ' + settings.className;
   }
 
-  var preserveDrawingBuffer = Boolean(settings.preserveDrawingBuffer);
-  this.gl = canvas.getContext('webgl',{preserveDrawingBuffer}) || canvas.getContext('experimental-webgl',{preserveDrawingBuffer});
+  this.gl = canvas.getContext('webgl') || canvas.getContext('experimental-webgl');
```

Comment:
oops, am I basing my work off an old copy? I can rebase and remove this. I guess i based off the gh-pages file and my diffing exposed this... sorry, will clean up!